Photino.Blazor/PhotinoWebViewManager.cs

```diff
@@ -80,7 +83,8 @@ protected override void NavigateCore(Uri absoluteUri)
 
         protected override void SendMessage(string message)
         {
-            Task.Run(() => Dispatcher.InvokeAsync(() => _window.SendWebMessage(message)));
+            Task.Run(() => { Dispatcher.InvokeAsync(() => _window.SendWebMessage(message)); })
+                .Wait(TimeSpan.FromSeconds(2));
```

Comment:
This 2 second delay seems a bit arbitrary / random..? 
Could you elaborate why and how you chose to add this delay here, specifically for 2 seconds? This could lead to some questionable delays for otherwise very fast message calls.

Reply:
Refer to Microsoft documentation https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.wait?view=net-6.0#system-threading-tasks-task-wait(system-timespan ).

The time span of the Wait function is timeout. 

In fact this task should wait until completion.

2 seconds just I think the Task may be done in this time

Reply:
Considering that some operations may be time-consuming, making other requests at this time will still cause the #40 problem. 
Please remove the timeout or set it to -1.